js/templates/listingCard.html

```diff
@@ -42,7 +42,10 @@
   </div>
   <% if (ob.vendor) { %>
   <a class="userIconWrapper js-userName">
-    <div class="userIcon disc clrBr2 clrSh1"
+    <% const vendorTip = `${ob.vendor.name}${ob.vendor.location ? `  📍  ${ob.vendor.location}` : ''}`;
```

Comment:
> As far as I can tell, it looks identical to the emojified version on Windows, we could probably leave this as-is, unless there's a big difference on other OSes.

@jjeffryes Keep in mind this also dependent on the version of an OS. If you go back far enough, those user will see jibberish (probably an empty rectangle).

So, @morebrownies, since we can't emojify (convert to an image) emojis in a tooltip, there are a couple options.
1.) Don't use emojis at all in tooltips.
2.) Use emoji's, but don't emojify them. The risk with this is that the emojis will look different across different OSs or look completely broken on non-supporting OSs.